util/downloadBuild.py

```diff
@@ -245,7 +246,7 @@ def downloadBuild(httpDir, targetDir, jsShell=False, wantSymbols=True, wantTests
                     localfile = os.path.join(buildDir, filename)
                     downloadURL(remoteURL, localfile, quiet=True)
                 gotApp = True
-            if remotefn.endswith('.mac.dmg') or remotefn.endswith('.mac64.dmg'):
+            elif re.search(r'(\.mac(64)?\.|^target)\.dmg$', fn):
```

Comment:
This incorrectly expects two consecutive dots. I think there shouldn't be a `\.` between `(64)?` and `|`.